### PR TITLE
Homogenize repo with organization

### PR DESCRIPTION
**Pull request by fcollonval**
Please finish the following when submitting a pull request:

- [x] Add a line to `History.md` briefly documenting the change.

If this is a release, additionally do the following:

- [x] Update the dependencies in `package.json`
- [x] Run `jlpm install` to update `yarn.lock`

Thanks!

### CODE REVIEW

File: style/index.css [hunk 1]
```diff
@@ -0,0 +1 @@
+@import url('base.css');
```
Comment (axelfahy) [inline style/index.css:1]: What is the reason for importing an empty file?

**Review by axelfahy — COMMENTED** [review]
Thank you for the homogenization of the repo with the other extensions.

File: README.md [hunk 2]
```diff
@@ -2,9 +2,12 @@
 
 > **Disclaimer**: fork of <https://github.com/jwkvam/jupyterlab-vim> for personal use. Use at your own risk.
 
-[![npm version](https://badge.fury.io/js/jupyterlab_vim.svg)](https://www.npmjs.com/package/jupyterlab_vim)
+
 [![Build Status](https://travis-ci.org/jwkvam/jupyterlab-vim.svg?branch=master)](https://travis-ci.org/jwkvam/jupyterlab-vim)
```
[the remark this answers is not in the document]
Comment (fcollonval) [reply README.md:6]: Thanks I did not catch that one.
I'll update it with the GitHub badge. I'll also merge the build.yml and the pull_request.yml - I did not look at them before but they are redundant.

File: style/index.css (hunk 1 above)
Comment (fcollonval) [reply style/index.css:1]: Thanks for catching this was a consequence of applying the upgrade extension script. But in this case this is indeed not needed.

**Review by axelfahy — APPROVED** [review]

**Pull request merged**